packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -627,16 +616,18 @@ export class SDKClient {
       } else {
```

Comment:
QQ: Shouldn't we set the status to success also in the if-block above after successfully executing the query?

```suggestion
        status = Status.Success.toString();
      } else {
```

Reply:
Ah yeah good catch! I actually moved it outside of the if/else block as it has the same syntax.